src/global-table.ts

```diff
@@ -77,12 +77,18 @@ abstract class GlobalTableBase extends Resource implements IGlobalTable {
 
 export interface GlobalTableProps {
   readonly partitionKey: ddb.Attribute;
+  readonly replicas?: Replica[];
   readonly tableName?: string;
 }
 
+export interface Replica {
+  readonly region: string;
+}
+
 export class GlobalTable extends GlobalTableBase {
   public readonly tableArn: string;
   public readonly tableName: string;
+  public readonly replicateregions: Replica[] = [];
```

Comment:
Why make this public? I don't think we need to expose this to consumers of the construct. 

And why not use the name `replicas`? This structure should hold more information than just the region in the future.